src/date.ml

```diff
@@ -476,7 +476,7 @@ value string_of_ondate conf d =
 value string_of_date conf =
   fun
   [ Dgreg d _ -> string_of_dmy conf d
-  | Dtext t -> "(" ^ t ^ ")" ]
+  | Dtext t -> t ]
```

Comment:
If this is the expected behavior for now, I suspect there may be other places where the code should be modified.